go.mod

```diff
@@ -25,8 +24,7 @@ require (
 	github.com/hashicorp/errwrap v1.0.0 // indirect
 	github.com/klauspost/compress v1.17.0 // indirect
 	github.com/leodido/go-urn v1.2.4 // indirect
-	github.com/minio/highwayhash v1.0.2 // indirect
-	github.com/nats-io/jwt/v2 v2.5.2 // indirect
+	github.com/nats-io/nats-server/v2 v2.10.2 // indirect
```

Comment:
NATS server still include, but now indirectly due to one of the other NATS dependencies?

Reply:
I just updated to latest nats.go client and force pushed @lenny-intel - this removed the dependency entirely.